audformat/core/utils.py

```diff
@@ -419,8 +419,10 @@ def to_segmented_index(
             :ref:`table specifications <data-tables:Tables>`
 
     """
-    if obj.empty or index_type(obj) == define.IndexType.SEGMENTED:
+    if index_type(obj) == define.IndexType.SEGMENTED:
         return obj
+    elif obj.empty:
+        return segmented_index()
```

Comment:
Why this change to an existing function?

Reply:
This fixes a bug we had - before when you passed an empty `filewise` index it was returning it instead of an empty `segmented` index.

Reply:
OK, can you then also update the test for `to_segmented_index()` to cover this error.

Reply:
done